src/openms/source/ANALYSIS/ID/IDScoreGetterSetter.cpp

```diff
@@ -68,7 +68,60 @@ namespace OpenMS
     }
   }
 
+  inline bool isFirstBetterScore_(double first, double second, bool isHigherBetter)
```

Comment:
I think inline doesn't make sense here as it is in the cpp file and we don't need to control linkage of the function.